### ISSUE TITLE
Home page block title contrast issue

### ISSUE THREAD

**Issue by dajoturner** [ucsc/wcms-design-templates#67]
Len Burns reported this issue. "The main page has table/boxes with info with headings such as "OUR TRAVELS", "FOR STUDENTS", "SUPPORT HISTORY" but the first word of each is in a lighter font for some stylistic reason. This also can be difficult to read.  The thinking behind WCAG 2.0 is that a person with uncorrected 20-40 vision should be able to read a web page at a comfortable distance from the monitor.  This is an over simplification, but gives a flavor for the intent."
Design element, out of scope for current project.
